Source/Libraries/CorruptCore/Extensions/ByteArrayExtensions.cs

```diff
@@ -126,6 +127,7 @@ private static BigInteger Mod(BigInteger x, BigInteger m)
             return (x % m + m) % m;
         }
 
+        [SuppressMessage("Microsoft.Design", "CA1062", Justification = "https://github.com/dotnet/roslyn-analyzers/issues/3451")]
```

Comment:
Is this the intended justification? Doesn't seem to completely explain things unless I'm missing something.

Reply:
Yes that was my intended justification, but no it doesn't completely explain things.

Suppressing this warning should be covered by the `.editorconfig` configuration I added. Unfortunately, the `.editorconfig` configuration is broken. So, we need to manually suppress each instance until that bug is fixed.

I can find a concise way to phrase the above to put in the message. I could also create an RTCV bug for the explanation and link to that?

Reply:
Yeah that'd make it a bit more clear for future reference rather than a large issue where it's hiding a bit down the chain.

Reply:
Done, #187